### DISCUSSION TITLE
Autosave in local settings

### DISCUSSION THREAD

**Discussion by mike-ward** [zed-industries/zed#13285] [Feature Requests]
I have the following in the local settings for my project
```
{
  "autosave": { "after_delay": {"milliseconds": 500} },
  "format_on_save": "on",
  "formatter": {
    "external": {
      "command": "v",
      "arguments": ["fmt"]
    }
  }
}
```
The `format_on_save` works but not the `autosave`. Do I need to include something extra?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
